CSLabs.Api/Email/SESSender.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon;
+using System.Linq;
+using Amazon.SimpleEmailV2;
+using Amazon.SimpleEmailV2.Model;
+using System.Threading;
+using FluentEmail.Core;
+using FluentEmail.Core.Interfaces;
+using FluentEmail.Core.Models;
+
+namespace CSLabs.Api.Email
+{
+    public class SESSender : ISender
+    {
+        private readonly string _accessKey;
+        private readonly string _secretKey;
+        private readonly RegionEndpoint _endpoint;
+
+        public SESSender(string accessKey, string secretKey, RegionEndpoint endpoint)
+        {
+            _accessKey = accessKey;
+            _secretKey = secretKey;
+            _endpoint = endpoint;
+        }
+        
+        public SendResponse Send(IFluentEmail email, CancellationToken? token = null)
+        {
+            return SendAsync(email, token).GetAwaiter().GetResult();
```

Comment:
I think to safely let this code run in different contexts we should add .ConfigureAwait(false). Definitely test it to make sure it still works however.

```suggestion
            return SendAsync(email, token).ConfigureAwait(false).GetAwaiter().GetResult();
```